specs/mount.go

```diff
@@ -0,0 +1,92 @@
+package specs
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/validate"
+)
+
+/*Mount mount
+
+swagger:model Mount
+*/
+type Mount struct {
+
+	/* destination
+
+	Required: true
+	*/
+	Destination FilePath `json:"destination"`
+
+	/* options
+	 */
+	Options ArrayOfStrings `json:"options,omitempty"`
+
+	/* source
+
+	Required: true
+	*/
+	Source FilePath `json:"source"`
+
+	/* type
+
+	Required: true
+	*/
+	Type *string `json:"type"`
+}
+
+// Validate validates this mount
+func (m *Mount) Validate(formats strfmt.Registry) error {
```

Comment:
there's a flag not to generate validation at all - not sure how to generate _only_ validation, but really, should we go ahead with this way of generating code with swagger?!